_rest_routes.inc.php

```diff
@@ -6234,7 +6237,247 @@
+     *          description="The message of the transaction.",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="type",
+     *          description="The type of transaction. Use an option from resource=/api/transaction_type",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="groupname",
+     *          description="The group name (usually is 'Default').",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="referByNpi",
+     *          description="NPI of the person creating the referral.",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="referToNpi",
+     *          description="NPI of the person getting the referral.",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="referDiagnosis",
+     *          description="The referral diagnosis.",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="riskLevel",
+     *          description="The risk level. (Low, Medium, High)",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="includeVitals",
+     *          description="Are vitals included (0,1)",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="referralDate",
+     *          description="The date of the referral",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="authorization",
+     *          description="The authorization for the referral",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="visits",
+     *          description="The number of vists for the referral",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="validFrom",
+     *          description="The date the referral is valid from",
+     *          type="string"
+     *      ),
+     *      @OA\Property(
+     *          property="validThrough",
+     *          description="The date the referral is valid through",
+     *          type="string"
+     *      ),
+     *      required={"message", "groupname", "title"},
+     *      example={
+     *          "message": "Message",
+     *          "type": "LBTref",
+     *          "groupname": "Default",
+     *          "referByNpi":"9999999999",
+     *          "referToNpi":"9999999999",
+     *          "referDiagnosis":"Diag 1",
+     *          "riskLevel":"Low",
+     *          "includeVitals":"1",
+     *          "referralDate":"2022-01-01",
+     *          "authorization":"Auth_123",
+     *          "visits": "1",
+     *          "validFrom": "2022-01-02",
+     *          "validThrough": "2022-01-03"
+     *      }
+     *  )
+     */
```

Comment:
to keep these comments as clean/organized as possible, this schema should go directly above first OA\* block that uses it, which is the `@OA\Post(` code block